testsuite/integration-arquillian/tests/base/src/test/resources/testrealm.json

```diff
@@ -414,7 +414,77 @@
       "enabled": true,
       "directAccessGrantsEnabled": true,
       "secret": "password",
-      "webOrigins": [ "http://localtest.me:8180" ]
+      "webOrigins": [ "http://localtest.me:8180" ],
+      "protocolMappers": [
+        {
+          "name": "aud-account",
+          "protocol": "openid-connect",
+          "protocolMapper": "oidc-audience-mapper",
+          "config": {
+            "included.client.audience": "account",
+            "id.token.claim": "true",
+            "access.token.claim": "true"
+          }
+        },
+        {
+          "name": "aud-admin",
+          "protocol": "openid-connect",
+          "protocolMapper": "oidc-audience-mapper",
+          "config": {
+            "included.client.audience": "security-admin-console",
+            "id.token.claim": "true",
+            "access.token.claim": "true"
+          }
```

Comment:
The hardcoded audiences have to be here to properly test permissions. Without this, the client has only "audience resolve" mapper which adds audience only when user has a role from given client. E.g. if we want to test that user cannot access some endpoint without some role, we need a hardcoded audience because without it the `aud` claim would be completely missing and the access would be denied not because the lack of required role but because of the missing `aud` claim.